app/components/admin/custom_fields/hierarchy/items_component.html.erb

```diff
@@ -32,7 +32,10 @@ See COPYRIGHT and LICENSE files for more details.
     flex_layout do |container|
       if items.empty? && !show_new_item_form?
         container.with_row(mb: 3) do
-          render Primer::Beta::Blankslate.new(border: true) do |component|
+          render Primer::Beta::Blankslate.new(
+            border: true,
+            test_selector: "op-custom-fields--hierarchy-items-blankslate"
```

Comment:
Honestly there is no good reason to extract those inline selectors to variables. I did it for the dialog, because it was already in the template I copied.

But those selectors are unique. They are not reused.